cogs/strike.py

```diff
@@ -557,14 +548,18 @@ async def _confirm_manual_add_strike(self, strike_user: discord.User | discord.M
                     )
                     and interaction.channel == confirmation_message_channel
                     and "custom_id" in interaction.data
-                    and interaction.data["custom_id"] in {
+                    and interaction.data["custom_id"]
+                    in {
```

Comment:
Comparison statements should not be split between lines. See above comment for further reasoning.